docs/source/index.rst

```diff
@@ -34,22 +33,30 @@ our `Slack`_ channel!
 
 Composer is part of the broader Machine Learning community, and we welcome any contributions, pull requests, or issues.
 
-Table of Contents
------------------
-
 .. toctree::
+   :hidden:
    :maxdepth: 1
    :caption: Getting Started
 
    getting_started/installation.rst
-   getting_started/using_composer.rst
+   getting_started/notebooks.rst
    getting_started/welcome_tour.rst
 
 .. toctree::
+   :hidden:
+   :maxdepth: 1
+   :caption: Algorithms
+
+   trainer/algorithms.rst
+   functional_api.rst
```

Comment:
Little confusing having Algorithms as a heading under Algorithms. Also we call them methods in the "Methods library" +  "Methods Overview" headings. Maybe one of...

- Using speedup methods
- Speedup methods
- Speedup method API

I also kind of like having the sub-headings be "Algorithm API" and "Functional API" instead of just "Algorithms" and "Functional".

But I definitely don't feel like I know the best answer here.